packages/jsActions/nanoflow-actions-native/package.json

```diff
@@ -25,13 +25,14 @@
   "dependencies": {
```

Comment:
Please bump this package's version and update the changelog

Reply:
done